@prefix dcterms: <http://purl.org/dc/terms/> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix schema: <https://schema.org/> .
@prefix sh: <http://www.w3.org/ns/shacl#> .

<https://www.wikidata.org#query-c146eaa74fb054c54e90b96dae932144> a sh:SPARQLExecutable,
    sh:SPARQLSelectExecutable;
  dcterms:isPartOf <https://www.wikidata.org/wiki/Wikidata:Contact_the_development_team/Archive/2019/10>;
  dcterms:license <https://creativecommons.org/licenses/by-sa/4.0/>;
  rdfs:comment "Query no longer working of ListeriaBot updates). Now, there is a systematic timeout error. Could you have a look at it, please? historyThe following query was working fine until recently (see"@en;
  sh:prefixes <https://example.org/to_decide/wikidata_prefixes>;
  sh:select """PREFIX wdt: <http://www.wikidata.org/prop/direct/>
PREFIX wd: <http://www.wikidata.org/entity/>
SELECT (wd:Q5 as ?item)(COUNT(DISTINCT ?item) as ?count)
  WHERE {
    hint:Query hint:optimizer \"None\".
    ?item wdt:P131+ wd:Q142.
    ?item (wdt:P31/wdt:P279*) wd:Q16970
  MINUS { ?item wdt:P625 [].}""";
  <https://purl.expasy.org/sparql-examples/ontology#bigdata_query> """SELECT (wd:Q5 as ?item)(COUNT(DISTINCT ?item) as ?count)
  WHERE {
    hint:Query hint:optimizer \"None\".
    ?item wdt:P131+ wd:Q142.
    ?item (wdt:P31/wdt:P279*) wd:Q16970
  MINUS { ?item wdt:P625 [].}""";
  schema:target <https://query.wikidata.org/sparql> .
